invenio_rdm_records/fixtures/data/vocabularies/relation_types.yaml

```diff
@@ -46,6 +46,18 @@
   title:
      en: Describes
      de: Beschreibt
+- id: hasmetadata
+  props:
+     datacite: HasMetadata
+  title:
+    en: Has metadata
+    de: Hat metadaten
+- id: ismetadatafor
+  props:
+     datacite: IsMetadataFor
+  title:
+    en: Is metadata for
+    de: Sind metadaten für
```

Comment:
`de: Hat Metadaten für`

additional information for translation:
datacite example: indicates additional metadata A for a resource B